CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ OpenTelemetry Go Automatic Instrumentation adheres to [Semantic Versioning](http
 
 ### Added
 
+- Manual instrumentation support - Phase 1 ([#523]https://github.com/open-telemetry/opentelemetry-go-instrumentation/pull/523)
```

Comment:
This needs to be updated to describe to the end user the functionality added. Ideally it will include the CLI flag when added and describe the details of its use.

Reply:
Updated